### ISSUE TITLE
Error when checking out the source

### ISSUE THREAD

**Issue by MartinDelille** [liberapay/liberapay.com#397]
I tried a fresh clone without success:

```
$ git clone git@github.com:liberapay/liberapay.com.git
Cloning into 'liberapay.com'...
remote: Counting objects: 54058, done.
remote: Total 54058 (delta 0), reused 0 (delta 0), pack-reused 54058
Receiving objects: 100% (54058/54058), 24.21 MiB | 1.71 MiB/s, done.
Resolving deltas: 100% (36427/36427), done.
Checking connectivity... done.
Downloading www/assets/mangopay/Mangopay_Terms-AT.pdf (819.09 KB)
Error downloading object: www/assets/mangopay/Mangopay_Terms-AT.pdf (4bd21269a96fa217ec75452fed617c50b5d66ba99d83743baabcbd9c9b896638)

Errors logged to /Users/martin/dev/Liberapay/temp/liberapay.com/.git/lfs/objects/logs/20160816T024548.029415249.log
Use `git lfs logs last` to view the log.
error: external filter git lfs smudge %f failed 2
error: external filter git lfs smudge %f failed
fatal: www/assets/mangopay/Mangopay_Terms-AT.pdf: smudge filter lfs failed
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'
```

I tried several workaround explained [here](https://github.com/github/git-lfs/issues/911) without success.

**Comment by MartinDelille** [comment]
I tried to contact directly github about the problem.

**Comment by Changaco** [comment]
Our [.lfsconfig](https://github.com/liberapay/liberapay.com/blob/129/.lfsconfig) file is confusing it. A temporary workaround is `git reset && git rm .lfsconfig && git commit -m XXX && git checkout -f HEAD && git reset --hard HEAD~`.

**Comment by MartinDelille** [comment]
Thanks!

**Comment by MartinDelille** [comment]
If we are using git lfs only for Mangopay pdfs, maybe we could ask for markdown version from them instead.

**Comment by MartinDelille** [comment]
Or just link a permanent URL.

**Comment by MartinDelille** [comment]
Removing the `.lfsconfig` file seems to fix the issue: 845bbcc

According to github staff, lfs config can be automaticaly detect from the remote url.
